x-pack/plugins/task_manager/server/task_claimers/strategy_mget.ts

```diff
@@ -317,14 +321,14 @@ async function searchAvailableTasks({
   definitions,
   taskTypes,
   removedTypes,
-  excludedTypes,
+  excludedTaskTypes,
   taskStore,
   size,
   taskPartitioner,
 }: OwnershipClaimingOpts): Promise<SearchAvailableTasksResponse> {
   const searchedTypes = Array.from(taskTypes)
     .concat(Array.from(removedTypes))
-    .filter((type) => !excludedTypes.has(type));
+    .filter((type) => !isTaskTypeExcluded(excludedTaskTypes, type));
```

Comment:
this allows us to do wildcard exclusions like `alerting*` instead of specifying the full task type. this is the same as what the default task claimer is doing

Reply:
thanks for fixing this bug!